windows-terminal-quake/Settings.cs

```diff
@@ -146,6 +146,8 @@ public class SettingsDto
 		public bool AlwaysOnTop { get; set; } = false;
 
 		public bool StartHidden { get; set; } = false;
+		
+		public List<string> IgnoreHotKeyWindows { get; set; }
```

Comment:
I'm looking for a somewhat more descriptive term for this setting, perhaps "SuppressHotKeyForProcesses"?

Reply:
Agreed, I'll update this!